src/test/java/uk/gov/digital/ho/hocs/info/api/ExtractServiceTest.java

```diff
@@ -24,7 +24,7 @@
     private SchemaService schemaService;
 
     private ExtractService service;
-    private final Field field = new Field("", "Field1", "", "", "", true);
+    private final Field field = new Field("", "Field1", "", "", "", true, null);
```

Comment:
It doesn't look like this is actually used. I'm not too bothered but it's a quick clean up to delete rather than amend.

Reply:
I've removed this.